components/gc-chckbxrdio/_base.scss

```diff
@@ -38,14 +38,8 @@ fieldset.gc-chckbxrdio {
 
 	input[type="radio"],
 	input[type="checkbox"] {
-		border: 0;
-		clip: rect(0 0 0 0);
-		height: 1px;
-		margin: -1px;
-		overflow: hidden;
-		padding: 0;
-		position: absolute;
-		width: 1px;
+		opacity: 0;
+		z-index: 2;
```

Comment:
@Christopher-O Was there a reason for this change? It makes it possible for users to select the checkbox when clicking on the white space beside the checkbox. This could confuse users.

Reply:
It's also to allow Dragon Naturally Speak users to select these items.  The current design does not allow users to select the checkbox or radio buttons.   All it does is increase the clickable area but it still close to each item so it should not be an issue.  If you would prefer I can update the positioning of the inputs when vertically aligned so they are more centred within the clickable area.

Reply:
I would add `margin-left: 0;`. Right now, when inside a `.checkbox` element, it adds a `margin-left: -20px` to the input.

Reply:
I added margin-left: 10px, as margin-left: 0px did not have the input on top of the small checkbox design, so this way it the input is now inside the enlarged design for both sizes.

Reply:
@Christopher-O Hmmm, it messes up the hover styling since it is over the large checkbox. I don't think we should impede on the user experience of the vast majority simply because one software is not able to do its job. I would suggest to revert those changes unless there is a way to place it behind the large checkbox and still being accessible by Dragon Naturally Speak.

@duboisp Would you like to chime in?

Reply:
I have pushed a fix to support this new checkbox position without impacting the hover behaviour. I have also added the documentation. Thank you for your contribution @Christopher-O !